extensions/PhabBugz/lib/WebService.pm

```diff
@@ -138,6 +138,102 @@ sub check_user_permission_for_bug {
     };
 }
 
+sub update_reviewer_statuses {
+    my ($self, $params) = @_;
```

Comment:
I don't see where we're doing the work of updating r+ flags.  This is still coming, I assume.